tests/profiles.spec.ts

```diff
@@ -37,9 +156,7 @@ async function createProfile(page: Page, profile: string) {
 }
 
 async function deleteProfile(page: Page, profile: string) {
```

Comment:
Shouldn't this and all the functions below be moved to a `profile-helpers` file, just like for the instances?